wrappers/swift/SDWLogConfiguration.swift

```diff
@@ -0,0 +1,208 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Reresents general logging properties.
+public class LogConfiguration {
+    private var odwLogConfiguration: ODWLogConfiguration
+
+    public init(withConfig config: ODWLogConfiguration) {
+        self.odwLogConfiguration = config
+    }
+
+    /// Return a copy of the default configuration
+    public static func getLogConfigurationCopy() -> LogConfiguration? {
+        if let odwLogConfiguration = ODWLogConfiguration.getCopy() {
```

Comment:
How does this copy works ? If it is creating a shallow copy of [_wrappedConfiguration](https://github.com/microsoft/cpp_client_telemetry/blob/5a511c3e1e05d5c59fcfd025b4c507376705b998/wrappers/obj-c/ODWLogConfiguration.mm#L295), this mayn't be right.

Reply:
Sorry to unresolve it, can you please reply to this comment, as I don't see any change done for it.

Reply:
I thought i did reply to this but perhaps it was not saved.
Its actually creating a new object in objc by passing cpp configurations and returning the newly created object.
i have tried to do the similar thing here with swift.

Reply:
resolving this comment. please reopen if any concerns.